docs/source/spatialstats.rst

```diff
@@ -0,0 +1,157 @@
+.. _spatialstats:
+
+Spatial statistics
+==================
+
+Spatial statistics, also referred to as `geostatistics <https://en.wikipedia.org/wiki/Geostatistics>`_, are essential for the analysis of observations distributed in space.
+To analyze DEMs, ``xdem`` integrates spatial statistics tools specific to DEMs described in recent literature,
+in particular in `Rolstad et al. (2009) <https://doi.org/10.3189/002214309789470950>`_,
+`Dehecq et al. (2020) <https://doi.org/10.3389/feart.2020.566802>`_ and
+`Hugonnet et al. (2021) <https://doi.org/10.1038/s41586-021-03436-z>`_. The implementation of these methods relies
+partly on the package `scikit-gstat <https://mmaelicke.github.io/scikit-gstat/index.html>`_.
+
+The spatial statistics tools can be used to assess the precision of DEMs (see the definition of precision in :ref:`intro`).
+In particular, these tools help to:
+
+- account for non-stationarities of elevation measurement errors (e.g., varying precision of DEMs with terrain slope),
+- quantify the spatial correlation of measurement errors in DEMs (e.g., native spatial resolution, instrument noise),
+- estimate robust errors for observations integrated in space (e.g., average or sum of samples),
+- propagate errors between spatial ensembles at different scales (e.g., sum of glacier volume changes).
+
+.. contents:: Contents 
+   :local:
+
+Spatial statistics for DEM precision estimation
+***********************************************
+
+Assumptions for statistical inference in spatial statistics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Spatial statistics are valid if the variable of interest verifies `the assumption of second-order stationarity
+<https://www.aspexit.com/en/fundamental-assumptions-of-the-variogram-second-order-stationarity-intrinsic-stationarity-what-is-this-all-about/>`_.
+That is, if the three following assumptions are verified:
+
+1. The mean of the variable of interest is stationary in space, i.e. constant over sufficiently large areas,
+2. The variance of the variable of interest is stationary in space, i.e. constant over sufficiently large areas.
+3. The covariance between two observations only depends on the spatial distance between them, i.e. no other factor than this distance plays a role in the spatial correlation of measurement errors.
+
+A sufficiently large averaging area is an area expected to fit within the spatial domain studied.
+
+In other words, for a reliable analysis, the DEM should:
+
+1. Not contain systematic biases that do not average out over sufficiently large distances (e.g., shifts, tilts), but can contain pseudo-periodic biases (e.g., along-track undulations),
+2. Not contain measurement errors that vary significantly in space.
```

Comment:
This makes me think, it would be great to have a tool to test this in xdem. I guess a large mean and std filter that accounts for NaNs would do the job... Mmmmh more work for me!